### PR TITLE
Sage building a simple defi application on the celo blockchain using python

### PR DESCRIPTION
**Pull request by Southpaw0**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -2727,6 +2727,13 @@ const Users: User[] = [
     tags: ['celosage', 'intermediate', 'smartcontract', 'solidity'], 
 
   },
+  {
+    title: 'Building a Simple DeFi Application on the Celo Blockchain Using Python',
+    description: 'In this tutorial, we will build a simple decentralized finance (DeFi) application on the Celo blockchain using Python',
+    preview: require('./showcase/intermediate/building-a-simple-defi-application-on-the-celo-blockchain-using-python.png'),
+    website: 'blog/tutorials/building-a-simple-defi-application-on-the-celo-blockchain-using-python',
```
Comment (Julius170) [inline src/data-tutorials/users.tsx:2734]: Can you edit the name of your header image file here,  It ha to be the exact same as the image name you added in the intermediate folder... hence the cause of it's undeployment

File: blog/sage-building-a-simple-defi-application-on-the-celo-blockchain-using-python/sage-building-a-simple-defi-application-on-the-celo-blockchain-using-python.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+        balances[msg.sender] += rewardAmount;
+
+        // Reset the staking information for the user
+        delete staking[msg.sender];
+
+        emit Transfer(address(this), msg.sender, amount + rewardAmount);
+        emit Withdrawn(msg.sender, amount, rewardAmount);
+    }
+
+    function balanceOf(address _account) public view returns (uint256) {
```
Comment (Julius170) [inline blog/sage-building-a-simple-defi-application-on-the-celo-blockchain-using-python/sage-building-a-simple-defi-application-on-the-celo-blockchain-using-python.md:229]: function is missing the override keyword

File: blog/sage-building-a-simple-defi-application-on-the-celo-blockchain-using-python/sage-building-a-simple-defi-application-on-the-celo-blockchain-using-python.md [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+## Step 2: Write the Smart Contract
+
+Next, create the smart contract for DeFi with functionalities like withdrawal, staking, transferring, and approval of funds. 
+
+Create a file called deploy.py in the root directory of your project.
+
+deploy.py
+
+```python
+// SPDX-License-Identifier: MIT
```
Comment (Julius170) [inline blog/sage-building-a-simple-defi-application-on-the-celo-blockchain-using-python/sage-building-a-simple-defi-application-on-the-celo-blockchain-using-python.md:74]: Your smart contract isn't compiling has 5 errors, all of which requires the override keywords in some function to bee compilable

**Review by Julius170 — COMMENTED** [review]
Hi @Southpaw0 I've left some comments you can make changes on

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Pull request merged**